op-node/rollup/derive/l1_block_info.go

```diff
@@ -250,9 +284,19 @@ func isEcotoneButNotFirstBlock(rollupCfg *rollup.Config, l2BlockTime uint64) boo
 	return rollupCfg.IsEcotone(l2BlockTime) && !rollupCfg.IsEcotoneActivationBlock(l2BlockTime)
 }
 
+// isInteropButNotFirstBlock returns whether the specified block is subject to the Isthmus upgrade,
+// but is not the actiation block itself.
+func isInteropButNotFirstBlock(rollupCfg *rollup.Config, l2BlockTime uint64) bool {
+	return rollupCfg.IsInterop(l2BlockTime) && !rollupCfg.IsInteropActivationBlock(l2BlockTime)
+}
```

Comment:
Since we use the pre-interop L1 tx one last time during the upgrade block, we must disallow the deposit-txs from using the CrossL2Inbox during this block.
If the CrossL2Inbox does not exist yet, then it is safe, but we have to ensure that the spec and code puts any Interop upgrade-txs *after* the user deposits.